### PR TITLE
Backend scaffold

### PR DESCRIPTION
**Pull request by TMattLee**
Scaffold of routes used in my last project with most of the old project specific information stripped.

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    
+  </head>
+  <body>
+    test test test
+  </body>
+</html>
```
Comment (veredrec) [inline views/index.html:9]: Is this our main html file? Is views inside public or is it the same as public? Just to make sure I understand the structure :wink:

File: server/models/post.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+  postType:     String,        // LOST or FOUND .... later ADOPTION
+  petName:      String,
+  petBreed:     String,
+});
```
Comment (veredrec) [inline server/models/post.js:8]: Do we want to add here more fields? As `petColor`, `petImageUrl` etc?

File: server/models/user.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const postSchema = require( './post.js' );
+
+const user = new Schema({
+  userId:             String,
+  userDisplayName:    String,
+  posts:              [ postSchema ],
+});
```
Comment (veredrec) [inline server/models/user.js:9]: Same here - we can add `userLocation`, `userContactNum` etc?

File: views/index.html (hunk 1 above)
Comment (calebhypes) [reply views/index.html:9]: views would be the equivalent of our public folder. It appears that when setting up this scaffold Matt used the original file structure/naming. This is something we can change/adapt to fit the file structure we've discussed previously.

**Pull request merged**